test_runner/fixtures/utils.py

```diff
@@ -158,8 +156,8 @@ def get_scale_for_db(size_mb: int) -> int:
     return round(0.06689 * size_mb - 0.5)
 
 
-ATTACHMENT_NAME_REGEX = re.compile(
-    r".+\.log|.+\.stderr|.+\.stdout|.+\.filediff|.+\.metrics|flamegraph\.svg|regression\.diffs|.+\.html"
+ATTACHMENT_NAME_REGEX: re.Pattern = re.compile(  # type: ignore[type-arg]
+    r"(?:flamegraph\.svg|regression\.diffs|.+\.(?:log|stderr|stdout|filediff|metrics|html))"
```

Comment:
Why do we need extra outer parentheses? I'd say it's harder to read with them.

```suggestion
    r".+\.(?:log|stderr|stdout|filediff|metrics|html)|flamegraph\.svg|regression\.diffs"
```

Reply:
there are two file names and several file extensions. Therefore I decided to group them this way:

1. To have file names at the very beginning
2. And group the extensions separately

As you can see, the group won't be stored because it's turned off.

I can revert this change back if it brings more questions than answers

Reply:
accepted